test/e2e/kubernetes/pod/pod.go

```diff
@@ -250,29 +250,25 @@ func RunCommandMultipleTimes(podRunnerCmd podRunnerCmd, image, name, command str
 		podName := fmt.Sprintf("%s-%d", name, r.Intn(99999))
 		var p *Pod
 		var err error
-		p, err = podRunnerCmd(image, podName, "default", command, true, sleep, duration, timeout)
-
+		p, err = podRunnerCmd(image, podName, "default", command, true, sleep, 10*time.Second, timeout)
 		if err != nil {
 			return successfulAttempts, err
 		}
-		succeeded, err := p.WaitOnSucceeded(sleep, duration)
+		succeeded, err := p.WaitOnSucceeded(sleep, timeout)
```

Comment:
We don't want to apply the "command timeout" to this wait func. There are two things here:

- wait for the k8s api to schedule and run a pod to completion
- ensure that the pod container execution time is within expected timeouts

For the first one we want to be more tolerant of unexpected environmental conditions.